### ISSUE TITLE
[Suggestion] Copy Cloth components

### ISSUE THREAD

**Issue by Kretol** [rurre/PumkinsAvatarTools#38]
I noticed that the copy components function wasn't copying cloth components, and after a search of Discord I noticed it was only mentioned mid-2020 and that it would require a bit more work to get it implemented.  Therefore, to avoid any potential inadvertent bug reports, I figured I'd post this suggestion/request for whenever it might be possible to implement. :)
Thank you!

**Comment by rurre** [comment]
Yeah, I attempted to add support for it once but it didn't work out so well. 

I think I got a better understanding of how to copy stuff now so that might happen soonish.
